conjure-java-core/src/main/java/com/palantir/conjure/java/types/ErrorGenerator.java

```diff
@@ -172,12 +171,37 @@ private static JavaFile generateErrorTypesForNamespace(
                 })
                 .collect(Collectors.toList());
 
+        List<MethodSpec> isRemoteExceptionDefinitions = errorTypeDefinitions.stream()
+                .map(entry -> {
+                    String typeName = CaseFormat.UPPER_CAMEL.to(
+                            CaseFormat.UPPER_UNDERSCORE, entry.getErrorName().getName());
+                    String methodName = "is" + entry.getErrorName().getName();
```

Comment:
Some other naming ideas:
`is{Error}Error`
`exceptionIs{Error}`

Reply:
It seems overly verbose to repeat the error name in the method name. Then call sites looks like:
```java
MyConjureError.isMyConjureError(e);
```

Would be less verbose to just do something like (not suggesting this exact naming):
```java
MyConjureError.hasName(e);
```

Reply:
In your case MyConjureError is the namespace of the error definition, and there can be multiple errors defined in it.